### ISSUE TITLE
Remove PySide6-related test skips

### ISSUE THREAD

**Issue by mdickinson** [enthought/envisage#555]
We're currently skipping some tests in our Ubuntu GitHub Actions runs because of a PySide6 bug that triggers an end-of-process segfault. Those test skips should be removed once they're no longer needed. This requires:

- [x] A PySide 6.4.3 release (already happened)
- [x] EDM to have PySide 6.4.3 or later
- [x] Pyface fixes for compatibility with PySide6 6.4.x. (Fixes already made; now waiting on a Pyface release.)

xref: #476
xref: https://bugreports.qt.io/browse/PYSIDE-2254

**Comment by mdickinson** [comment]
#528 is related
